pom.xml

```diff
@@ -32,7 +32,7 @@
     <relativePath />
   </parent>
   <artifactId>xcode-plugin</artifactId>
-  <version>2.0.2-SNAPSHOT</version>
+  <version>2.0.3-SNAPSHOT</version>
```

Comment:
I am the one to blame on this one, but let's use 2.0.5 as the "new version" since our latest release is "2.0.4"

Reply:
:+1: fixed